--- zm-build/rpmconf/Build/get_plat_tag.sh	2025-02-27 15:00:48.978321658 +0100
+++ get_plat_tag.sh	2025-02-27 15:00:01.285969824 +0100
@@ -99,6 +99,18 @@
     exit 0
   fi
 
+  grep "AlmaLinux release 8" /etc/redhat-release > /dev/null 2>&1
+  if [ $? = 0 ]; then
+    echo "RHEL8${i}"
+    exit 0
+  fi
+
+  grep "AlmaLinux release 9" /etc/redhat-release > /dev/null 2>&1
+  if [ $? = 0 ]; then
+    echo "RHEL9${i}"
+    exit 0
+  fi
+
   grep "Fedora release 23" /etc/redhat-release >/dev/null 2>&1
   if [ $? = 0 ]; then
     echo "F23${i}"
