--- zm-web-client/WebRoot/js/zimbraMail/mail/view/ZmInviteMsgView.js	2025-03-21 20:06:24.299759393 +0100
+++ ZmInviteMsgView.js	2025-03-21 20:10:06.215783156 +0100
@@ -530,11 +530,11 @@
 	{
         var from = this._msg.getAddress(AjxEmailAddress.FROM) && this._msg.getAddress(AjxEmailAddress.FROM).getAddress();
         subs.counterInvMsg =  (!sentByAddr || sentByAddr == from) ?
-            AjxMessageFormat.format(ZmMsg.counterInviteMsg, [from]):AjxMessageFormat.format(ZmMsg.counterInviteMsgOnBehalfOf, [sentByAddr, from]);
+            AjxMessageFormat.format(ZmMsg.counterInviteMsg, [from]):AjxMessageFormat.format(ZmMsg.counterInviteMsgOnBehalfOf, [AjxStringUtil.htmlEncode(sentByAddr), from]);
 	}
 	// Fix for bug: 88052 and 77237. Display cancellation banner to organizer or attendee
 	else if (isInviteCancelled) {
-		var organizer = this._invite.getOrganizerName() || this._invite.getOrganizerEmail();
+                var organizer = AjxStringUtil.htmlEncode(this._invite.getOrganizerName() || this._invite.getOrganizerEmail());
 		subs.ptstMsg = AjxMessageFormat.format(ZmMsg.inviteMsgCancelled, organizer.split());
 		subs.ptstIcon = ZmCalItem.getParticipationStatusIcon(ZmCalBaseItem.PSTATUS_DECLINED);
 		subs.ptstClassName = "InviteStatusDecline";
