--- zm-web-client/WebRoot/js/zimbraMail/mail/view/ZmInviteMsgView.js	2025-07-04 21:47:37.424548473 +0200
+++ ZmInviteMsgView.js	2025-07-04 21:35:01.219253174 +0200
@@ -544,12 +544,12 @@
 		var attendee = this._invite.getAttendees()[0];
 		var ptst = attendee && attendee.ptst;
 		if (ptst) {
-            var names = [];
-			var dispName = attendee.d || attendee.a;
-            var sentBy = attendee.sentBy;
-            var ptstStr = null;
-            if (sentBy) names.push(attendee.sentBy);
-            names.push(dispName);
+			var names = [];
+			var dispName = AjxStringUtil.htmlEncode(attendee.d || attendee.a);
+			var sentBy = AjxStringUtil.htmlEncode(attendee.sentBy);
+			var ptstStr = null;
+			if (sentBy) names.push(AjxStringUtil.htmlEncode(attendee.sentBy));
+			names.push(dispName);
 			subs.ptstIcon = ZmCalItem.getParticipationStatusIcon(ptst);
 			switch (ptst) {
 				case ZmCalBaseItem.PSTATUS_ACCEPT:
